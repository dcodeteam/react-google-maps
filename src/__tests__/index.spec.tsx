import * as index from "../index";

describe("index", () => {
  it("should provide public api", () => {
    expect(index).toMatchSnapshot();
  });
});
