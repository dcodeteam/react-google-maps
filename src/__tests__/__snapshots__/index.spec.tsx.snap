// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should provide public api 1`] = `
Object {
  "CustomControl": [Function],
  "DataPolygon": [Function],
  "DrawingControl": [Function],
  "FitBounds": [Function],
  "FullscreenControl": [Function],
  "GoogleMap": [Function],
  "GoogleMapContextConsumer": [Function],
  "GoogleMapContextProvider": Object {
    "$$typeof": Symbol(react.provider),
    "_context": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": [Circular],
      "Provider": [Circular],
      "_calculateChangedBits": null,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {},
      "_currentValue2": Object {},
      "unstable_read": [Function],
    },
  },
  "InfoWindow": [Function],
  "MapComponent": [Function],
  "MapComponentHandlers": [Function],
  "MapTypeControl": [Function],
  "Marker": [Function],
  "MarkerContextConsumer": [Function],
  "MarkerContextProvider": Object {
    "$$typeof": Symbol(react.provider),
    "_context": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": [Circular],
      "Provider": [Circular],
      "_calculateChangedBits": null,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {},
      "_currentValue2": Object {},
      "unstable_read": [Function],
    },
  },
  "MarkerIcon": [Function],
  "MarkerSymbol": [Function],
  "PanBy": [Function],
  "PanTo": [Function],
  "PanToBounds": [Function],
  "Polyline": [Function],
  "RotateControl": [Function],
  "ScaleControl": [Function],
  "StreetViewControl": [Function],
  "ZoomControl": [Function],
}
`;
